docs/source/compilation/index.rst

```diff
@@ -0,0 +1,23 @@
+.. _compilation:
+
+Compilation
```

Comment:
I think "Building" would be a better title. Also, the number of `=` in the line below should be equal to the length of this one ;)